horovod/spark/common/store.py

```diff
@@ -532,23 +533,37 @@ def matches(cls, path):
         return path.startswith(cls.FS_PREFIX)
 
 
+# If `_DBFS_PATH_MAPPING_TO_PATH` is not None, map `/dbfs/...` path to `{_DBFS_PATH_MAPPING_TO_PATH}/...`
+# This is used in testing.
+_DBFS_PATH_MAPPING_TO_PATH = None
```

Comment:
Maybe the naming could be improved here, e.g. `_DBFS_PATH_PREFIX` or `_DBFS_PREFIX_MAPPING`.

The comment should also indicate that this mapping only applies to `get_localized_path`.